README.md

```diff
@@ -95,3 +95,11 @@ The following example scenarios are available:
 [Binding an Imported app to an Off-cluster Operator Managed IBM Cloud Service](examples/nodejs_ibmcloud_operator/README.md)
 
 [Binding an Imported app in one namespace with an In-cluster Managed PostgreSQL Database in another namespace](examples/nodejs_postgresql_namespaces/README.md)
+
+## Service Binding Meetings
```

Comment:
We can rename this to Community, discussion, contribution, and support, something more descriptive.

And we should also  add the mailing list that was created for any support/discussion

Reply:
@Avni-Sharma Let's rename the heading to `Service Binding Community`?

Reply:
This is README of Service Binding project already, hence no need to repeat "Service Binding", agree with @Avni-Sharma . +1 for  "Community, discussion, contribution, and support" .

Reply:
@pedjak What about mailing list info?
Should it be here or not?

Reply:
good idea, +1.

Reply:
Done